content/community/governance/gcb/index.md

```diff
@@ -0,0 +1,102 @@
+# Galaxy Community Board Proposal
```

Comment:
This section from "Community Special Interest Groups (SIGs)" through to the end of "Many hands make light work" seems like it should go under the 'SIGs' heading in the drop-down menu at the top, rather than on the GCB page.

Reply:
You could have a hook describing SIGs and embed it here (and on the SIGs page) but this page should be about the governance, not what a SIG is

Reply:
I guess I understand now. I intended that the SIG page is "just" listing SIGs and the description, duties, etc are part of the governance and hence part of this page.

The `hook` thing we need to implement into the framework. I hope we can do that soon.